test/core/Do.Test.Architecture/Extensions/IntegerExtensions.cs

```diff
@@ -0,0 +1,8 @@
+using Do.Testing;
+
+namespace Do.Test;
+
+public static class IntegerExtensions
```

Comment:
this should be coming from core extensions, check if it exists, if so remove this, if not move this to core